corehq/apps/geospatial/models.py

```diff
@@ -68,6 +79,7 @@ class GeoConfig(models.Model):
     api_token = models.CharField(max_length=255, blank=True, null=True, db_column="api_token")
 
     def save(self, *args, **kwargs):
+        assert self.travel_mode in [self.WALKING, self.CYCLING, self.DRIVING]
```

Comment:
isn't this already validated by the model since we add choices there?

Reply:
Unfortunately not; one is still able to create a record with a different value.

Reply:
okay. I believe the model supports adding validators as well in case you want to consider that but this is also okay

Reply:
That's true. I forgot about those. Maybe I'll add it there